Yoann-Abbes/The-count-of-money
# add dashboard scss files and bootstrap app

Replace infos in dedicated files
Add dashboard scss files
Bootstrap app with sidebar / appbar / chart

## How to test it

```cd frontend && npm i && npm run serve```

Go to localhost:8080 in your browser.
You should see a interface with topbar, sidebar, an rssflows page and a currencyDetails page with a candlestick chart. (all values displayed (sidebar/currencyDetails page) are fake and hard stored in front)
